meetup/services/generic_api.py

```diff
@@ -0,0 +1,21 @@
+from django.http import HttpResponse
```

Comment:
This looks unused, as does the import statement underneath it. Consider installing an editor plugin that detects and highlights unused dependencies.